src/config/dynamic-theme-fixes.config

```diff
@@ -17665,6 +17665,30 @@ a.logo[title="United Nations"]
 
 ================================
 
+understandingwar.org
+
+CSS
+#edit-search-block-form--2,
+#nice-menu-1 > li > a {
+    color: #665c39 !important;
```

Comment:
Could this be in the form of `color: ${#original hex color} !important`?

Reply:
I couldn't get that to work well because the original color is what is desired in this case. Overall a tricky site to work with and I think this is as clean as the patch can be unfortunately.

Reply:
I switched it out with `var(--darkreader-neutral-background)` which allows it to be readable and not have a hardcoded color. @Gusted

Reply:
> I couldn't get that to work because the original color is what is desired in this case.

You can take the output of `${original hex color}` and use that in the fix(it's a "perfect invert")